project.clj

```diff
@@ -70,6 +81,12 @@
                    ; SERVER-332, enable SSLv3 for unit tests that exercise SSLv3
                    :jvm-opts      ["-Djava.security.properties=./dev-resources/java.security"]}
 
+             :ezbake {:dependencies ^:replace [[puppetlabs/puppet-server ~ps-version]
+                                               [puppetlabs/trapperkeeper-webserver-jetty9 ~tk-jetty-version]
+                                               [org.clojure/tools.nrepl "0.2.3"]]
+                      :plugins [[puppetlabs/lein-ezbake "0.1.0"]]
```

Comment:
I *think* we may want to use `0.1.2` here instead to get the `net-tools` dependency:
https://github.com/puppetlabs/ezbake/commits/2be081f9c8033cce52ffd4e70218901060945511